src/System.Diagnostics.Tracing/src/System/Diagnostics/Tracing/TraceLogging/TraceLoggingTypeInfo.cs

```diff
@@ -61,7 +61,7 @@ internal TraceLoggingTypeInfo(Type dataType)
 
             if (name == null)
             {
-                throw new ArgumentNullException("eventName");
+                throw new ArgumentNullException(nameof(name));
```

Comment:
Let's revert this.  While this function's parameter is named "name", this is arrived at via many different entrypoints, some of which are named "eventName".